tests/common/plugins/fib.py

```diff
@@ -238,7 +238,11 @@ def fib_t1_lag(ptfhost, testbed, localhost):
             routes_v4 = generate_routes("v4", podset_number, tor_number, tor_subnet_number,
                                         None, leaf_asn_start, tor_asn_start,
                                         local_ip, local_ipv6, router_type="tor")
+            routes_v6 = generate_routes("v6", podset_number, tor_number, tor_subnet_number,
+                                        None, leaf_asn_start, tor_asn_start,
+                                        local_ip, local_ipv6, router_type="tor")
             announce_routes(ptfip, port, routes_v4)
+            announce_routes(ptfip, port6, routes_v6)
```

Comment:
can you combine 'spine' and 'tor' if blocks?

Reply:
Nice catch! Combined.